packages/extension/package.json

```
@@ -29,7 +29,7 @@
     "graphql": "^15.7.2",
     "graphql-request": "^3.6.1",
     "idb-keyval": "^5.1.5",
-    "next": "^12.2.2",
+    "next": "14.2.8",
```

```suggestion
    "next": "14.2.9",
```
New version just dropped